tests/QueryType/Select/Query/Component/Facet/PivotTest.php

```diff
@@ -95,48 +95,41 @@ public function testSetFields()
 
     public function testAddStat()
     {
-        $expectedStats = $this->facet->getStats();
+        $expectedStats = $this->facet->getLocalParameters()->getStats();
         $expectedStats[] = 'newstat';
-        $this->facet->addStat('newstat');
-        $this->assertSame($expectedStats, $this->facet->getStats());
+        $this->facet->getLocalParameters()->setStat('newstat');
+        $this->assertSame($expectedStats, $this->facet->getLocalParameters()->getStats());
     }
 
     public function testClearStats()
     {
-        $this->facet->addStat('newstat');
-        $this->facet->clearStats();
-        $this->assertSame([], $this->facet->getStats());
+        $this->facet->getLocalParameters()->setStat('newstat');
+        $this->facet->getLocalParameters()->clearStats();
+        $this->assertSame([], $this->facet->getLocalParameters()->getStats());
     }
 
     public function testAddStats()
     {
         $stats = ['stat1', 'stat2'];
 
-        $this->facet->clearStats();
-        $this->facet->addStats($stats);
-        $this->assertSame($stats, $this->facet->getStats());
-    }
-
-    public function testAddStatsAsStringWithTrim()
```

Comment:
i removed this test because i don't think we should support converting strings to array values based on an arbitrary explode character...